samples/nrf9160/nrf_cloud_rest_device_message/README.rst

```diff
@@ -0,0 +1,110 @@
+.. _nrf_cloud_rest_device_message:
+
+nRF9160: nRF Cloud REST Device Message
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The REST Device Message sample demonstrates how to use the `nRF Cloud REST API`_ to send a `Device Message <nRF Cloud Device Messages_>`_ using the ``SendDeviceMessage`` endpoint. The d2c topic is used in non-bulk mode.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+The sample requires an `nRF Cloud`_ account.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later.
+
+Overview
+********
+
+Pressing the button sends a button press message that you can view in the Terminal card of the `nRF Cloud`_ portal (check the Device-to-Cloud channel).
```

Comment:
This makes the reader wonder which button and did they need to configure something first.
What does the Device-to-Cloud channel refer to?
Should we start from the prerequisites first? Something like:
"To start working with this sample, make sure your device is provisioned (connected to cloud). If not, enable the nnn Kconfig option and follow the instructions..."